build/dependencies.props

```diff
@@ -1,24 +1,25 @@
 ﻿<Project>
   <PropertyGroup>
-    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <AspNetCoreVersion>2.0.0</AspNetCoreVersion>
     <BenchmarkDotNetVersion>0.10.3</BenchmarkDotNetVersion>
-    <CoreFxVersion>4.4.0-*</CoreFxVersion>
-    <DiagnosticSourceVersion>4.4.1-*</DiagnosticSourceVersion>
-    <DependencyModelVersion>2.0.0-*</DependencyModelVersion>
-    <InternalAspNetCoreSdkVersion>2.0.1-*</InternalAspNetCoreSdkVersion>
+    <CoreFxVersion>4.4.0</CoreFxVersion>
+    <DiagnosticSourceVersion>4.4.1</DiagnosticSourceVersion>
+    <DependencyModelVersion>2.0.0</DependencyModelVersion>
+    <InternalAspNetCoreSdkVersion>2.0.1-rtm-15400</InternalAspNetCoreSdkVersion>
     <JsonNetVersion>10.0.1</JsonNetVersion>
     <MoqVersion>4.7.49</MoqVersion>
-    <NETStandardImplicitPackageVersion>2.0.0-*</NETStandardImplicitPackageVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
-    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
+    <NETStandardImplicitPackageVersion>2.0.0</NETStandardImplicitPackageVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0</NETStandardLibraryNETFrameworkVersion>
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0</RuntimeFrameworkVersion>
     <RoslynVersion>2.3.1</RoslynVersion>
-    <RoslynDevVersion>2.6.0-beta*</RoslynDevVersion>
+    <RoslynDevVersion>2.6.0-beta1-61924-08</RoslynDevVersion>
     <StreamJsonRpcVersion>1.1.92</StreamJsonRpcVersion>
-    <TestSdkVersion>15.3.0-*</TestSdkVersion>
+    <TestSdkVersion>15.3.0</TestSdkVersion>
 
     <!-- Must keep in sync with tooling\Microsoft.VisualStudio.RazorExtension\project.json -->
     <VsShellVersion>15.0.26606</VsShellVersion>
     
-    <XunitVersion>2.3.0-beta2-*</XunitVersion>
+    <XunitVersion>2.3.0-beta2-build3683</XunitVersion>
+    <XunitRunnerVisualStudioVersion>2.3.0-beta2-build1317</XunitRunnerVisualStudioVersion>
```

Comment:
These are two different versions. Had to add these to distinguish between the two packages we reference. The remainder are pretty straighforward.